### DISCUSSION TITLE
Difference between DO and d0.

### DISCUSSION THREAD

**Discussion by alcobrake** [Bodmer/TFT_eSPI#3363] [Q&A - Touch]
Hi Bodmer,
i have doubt on the defining the macro which is used for working ILI9488 by using tftf_eSPI library.
while i am deafining the D0(Dzero) the macro working fine in 'user_setup' file. But while defining the macro as DO(Dout) the macro is not wrking.
may i know the reason behind.
But if the DIN pin exists there should be DOUT has to be there. But there is no DOUT there is D0.
Please solve my issuse.
Thank you.

**Comment by Bodmer** [comment]
See example setup files. Summary of potential TFT signal pin names below:
```
//#define TFT_MISO 19  // TFT pin may be labelled SDO, DO, DOUT, MISO etc.

//#define TFT_MOSI 23  // TFT pin may be labelled SDI, DI, DIN, MOSI etc.

//#define TFT_SCLK 18  // SPI Clock pin, TFT pin may be labelled SCLK, SCK etc.

//#define TFT_CS   15  // Chip Select control, TFT pin may be labelled CS, SS etc.

//#define TFT_DC    2  // Data Command control, TFT pin may be labelled DC, RS, A0 etc.

//#define TFT_RST   4  // Reset pin (could connect to RST pin), TFT pin may be labelled RST, RESET etc.

//#define TFT_RST  -1  // Set TFT_RST to -1 if display RESET is connected to ESP32 board RST
```
If you invent macros then the library may not recognise them.
